### DISCUSSION TITLE
Backup Name format

### DISCUSSION THREAD

**Discussion by GrahamWillsher** [lavinir/hassio-onedrive-backup#74] [Ideas]
Hi,

Would it be possible to be able to format the name of the back up to use fields (hour,minute,seconds, Day, date) so that we can format the day/date in our locale format?

E.g. for me it would be DD-MM-YYYY. rather than the current YYYY-MM-DD

Thanks,
Graham.

**Comment by lavinir** [comment]
I don't think I'd go as far as having an additional data format as input in the config but I'll look into using the relevant format for the configured locale
